CHANGELOG.md

```diff
@@ -0,0 +1,88 @@
+# Changelog
+
+## 3.0.0
```

Comment:
I am a bit confused with the change log versions.
The latest version before this PR is 2.1.0 but here we have documented 2.2.0, 3.0.0, 3.0.1 and then 3.0.0 again. Is there a typo here somewhere?

Reply:
I have no idea where this changelog has come from to be honest. I think i may have copied it from a different repo by accident.

Reply:
Have updated it to a fresh one with just a 3.0.0 note.